sh/provision.sh

```diff
@@ -9,7 +9,9 @@ aws_region=${AWS_REGION} \
 aws_access_key=${AWS_ACCESS_KEY} \
 aws_secret_access_key=${AWS_SECRET_ACCESS_KEY} \
 cluster_name=${CLUSTER_NAME} \
-share_cluster_credentials=${SHARE_CLUSTER_CREDENTIALS} \
-platform=${PLATFORM} \
+cluster_environment=${CLUSTER_ENVIRONMENT} \
 environment_type=${ENVIRONMENT_TYPE} \
-cluster_environment=${CLUSTER_ENVIRONMENT} "
+konstructor_api_key=${KONSTRUCTOR_API_KEY} \
+oidc_issuerurl=${OIDC_ISSUER_URL} \
```

Comment:
Can you please rename `oidc_issuerurl` to `oidc_issuer_url` so it's consistent with the rest of the names?

Reply:
Done 👍